.travis.yml

```diff
@@ -1,102 +1,106 @@
-#
-# Operating Environment
-#
-language: rust
-sudo: false
+# Based on the "trust" template v0.1.1
+# https://github.com/japaric/trust/tree/v0.1.1
+
 dist: trusty
-services:
-  - docker
-addons:
-  apt:
-    packages:
-      - gcc-multilib
-      - libcurl4-openssl-dev
-      - libelf-dev
-      - libdw-dev
-      - binutils-dev
-
-rust:
-  - 1.7.0  # Oldest supported version
-  - stable
-  - beta
-  - nightly
-
-#
-# Environment Variables and Build Matrix
-#
+language: rust
+services: docker
+sudo: required
+
+# This is the Rust channel that build jobs will use by default but can be
+# overridden on a case by case basis down below
+rust: stable
+
 env:
   global:
-    - PATH=$HOME/.local/bin:$PATH
-    - TRAVIS_CARGO_NIGHTLY_FEATURE=""
-  matrix:
-    - ARCH=x86_64
-    - ARCH=i686
+    - CRATE_NAME=nix
 
-os:  # OSX included in build matrix explicitly
-  - linux
+    # default job
+    - TARGET=x86_64-unknown-linux-gnu
```

Comment:
Also, I think this should be done within the `matrix` section. If you look at the Travis output, you see `TARGET` being declared twice.